ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -332,7 +323,17 @@ private SafeFuture<BlockImportResult> handleBlockImport(
                     LOG.warn("Unable to import block {} due to invalid data", block.toLogString());
                     blobSidecarPool.removeAllForBlock(block.getRoot());
                     break;
-                  default:
+                  case FAILED_BROADCAST_VALIDATION:
+                    LOG.warn(
+                        "Unable to import block {} due to failed broadcast validation",
+                        block.toLogString());
+                    break;
```

Comment:
I have concerns that we could have some unresolved tails in this case. For example gossip validation failed because now parent, which means it's a future block and previously you would add it to the future blocks but with broadcast validation not. Could it be an issue (and may be other similar cases)?

Reply:
I think the key thing here is that we are talking about a locally produced block, so timing issues shouldn't apply. 
My initial idea was mainly avoid it to be put in the invalid cache, because the block it isn't actually invalid.

An interesting situation could be:

VC calls block publishing API via 2 different BN in parallel.
the first BN publishing it can cause the other to fail the GOSSIP validation rule. So in this case we really want to not consider the block invalid otherwise because the block has been actually sent. We want the node to be able to recover.

Can't think of other cases ATM

Reply:
Got you. Makes sense.